town.logic

```diff
@@ -1,2 +1,119 @@
 area Town:
-  room Main
\ No newline at end of file
+  room Main:
+    node Anju:
+      chest AnjuHP
+    node AnjuGarden:
+      door Anju
```

Comment:
You can leave Anju's garden by talking to her (and waiting to fail the minigame)
```suggestion
    node AnjuGarden:
      door Anju
    AnjuGarden -> Main # do we want to be able to make this out of logic (so as to not require this in entrance rando)?
```